helmfile/charts/rclone-sync/files/rclone.conf

```diff
@@ -12,10 +12,16 @@ v2_auth = {{ .s3.v2Auth | default false }}
 {{- end }}
 
 {{- if eq .type "swift" }}
+{{- if and (get .swift "applicationCredentialID") (get .swift "applicationCredentialSecret") }}
 application_credential_id = {{ .swift.applicationCredentialID }}
 application_credential_secret = {{ .swift.applicationCredentialSecret }}
+{{- else }}
+user = {{.swift.username }}
+key = {{.swift.password }}
```

Comment:
If username and password can now be used for sync shouldn't that be in [secrets.yaml](https://github.com/elastisys/compliantkubernetes-apps/blob/e4528a9aee84d6e40356f6030e891c3e24fc7477/config/secrets/sc-secrets.yaml#L16) and if so maybe specify that it can be either application credentials or username and password.

Reply:
added + comment